open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaClient.java

```diff
@@ -9,65 +9,190 @@
 
 import java.util.List;
 
+/**
+ * @param <T> object type for supplied and return.
+ * Interface describing common methods to client working with Subject area resources.
+ */
 public interface SubjectAreaClient<T> {
 
     FindRequest EMPTY_FIND_REQUEST = new FindRequest();
 
+    /**
+     * Get object by guid
```

Comment:
Can we change object to Node as this is more accurate

Reply:
In this case, there may be other objects (relationship for example and etc) and not only node.

Reply:
apologies for misreading. I suggest we say Node or Line instead of object,  then it is very explicit and we do not have to use the word object - which is very generic so does not add meaning.

Reply:
If you do not like Node or Line, instead of object. How about Glossary Artifact ?